edx_reverification_block/xblock/tests/test_reverification_block.py

```diff
@@ -188,10 +195,27 @@ def test_skip_reverification(self, xblock):
         # Reloading the student view, we should see that we've skipped
         self._assert_in_student_view(xblock, "skipped")
 
+    @ddt.data(0, 1, 2, 3, 4)
+    @scenario(TESTS_BASE_DIR + '/data/basic_scenario.xml', user_id='bob')
+    def test_grace_period_with_expired_due_date(self, xblock, grace_days):
+        # Verify that user can view the icrv for next 5 days even due data is expired.
+        # If due date is 1st Jan then user can view icrv till 5th Jan.
+
+        xblock.due = (datetime.datetime.today()-datetime.timedelta(grace_days)).replace(tzinfo=pytz.UTC)
```

Comment:
@clintonb I have reverted this change. Reason is utcnow() gives naive values. But in Xblock the self.due is offset aware. So It is giving error while comparison. Instead of changing the xblock code I reverted the change in test case.